bss/config.go

```diff
@@ -0,0 +1,87 @@
+package bss
+
+import (
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/urfave/cli"
+
+	"github.com/ethereum-optimism/optimistic-specs/bss/flags"
+)
+
+type Config struct {
+	/* Required Params */
+
+	// L1EthRpc is the HTTP provider URL for L1.
+	L1EthRpc string
+
+	// L2EthRpc is the HTTP provider URL for L1.
+	L2EthRpc string
+
+	// SROAddress is the SCC contract address.
+	SROAddress string
```

Comment:
oops, the `SCC` part was left over from when this was renamed from `SCCAddress`. i've replaced `SCC` with `SRO` since this matches the current name of the solidity file, but we can definitely change it when we agree on a new name :)